python/gtsam/examples/TranslationAveragingExample.py

```diff
@@ -0,0 +1,112 @@
+"""
+GTSAM Copyright 2010-2018, Georgia Tech Research Corporation,
+Atlanta, Georgia 30332-0415
+All Rights Reserved
+Authors: Frank Dellaert, et al. (see THANKS for the full author list)
+
+See LICENSE for the license information
+
+This example shows how 1dsfm uses outlier rejection (MFAS) and optimization (translation recovery)
+together for estimating global translations from relative translation directions and global rotations.
+The purpose of this example is to illustrate the connection between these two classes using a small SfM dataset.
+
+Author: Akshay Krishnan
+Date: September 2020
+"""
+
+import numpy as np
+
+import gtsam
+from gtsam.examples import SFMdata
+
+
+def get_data():
+    """"Returns data from SfMData.createPoses(). This contains global rotations and unit translations directions."""
+    # Using toy dataset in SfMdata for example.
+    poses = SFMdata.createPoses(gtsam.Cal3_S2(50.0, 50.0, 0.0, 50.0, 50.0))
+    rotations = gtsam.Values()
+    translation_directions = []
+    for i in range(0, len(poses) - 2):
+        # Add the rotation
+        rotations.insert(i, poses[i].rotation())
+        # Create unit translation measurements with next two poses
+        for j in range(i + 1, i + 3):
+            i_Z_j = gtsam.Unit3(poses[i].rotation().unrotate(
+                poses[j].translation() - poses[i].translation()))
+            translation_directions.append(gtsam.BinaryMeasurementUnit3(
+                i, j, i_Z_j, gtsam.noiseModel.Isotropic.Sigma(3, 0.01)))
+    # Add the last two rotations.
+    rotations.insert(len(poses) - 1, poses[-1].rotation())
+    rotations.insert(len(poses) - 2, poses[-2].rotation())
+    return (rotations, translation_directions)
+
+
+def estimate_poses_given_rot(measurements: gtsam.BinaryMeasurementsUnit3,
```

Comment:
i think the function name could be improved a bit more, since we estimate poses given rotations and directions, not just given rot

Reply:
I understand the confusion, but don't you think estimate_poses_from_rotations_and_unit_translations() is a bit too long? Even then it would not clarify what the unit translations are. Maybe I can change it to estimate_poses() and let the input arguments speak for the rest?

Reply:
`estimate_poses_from_rot_unitt`?

Reply:
or maybe just `estimate_poses()`?

Reply:
I updated it to `estimate_poses()`.